ts-schedule/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @topology_group main
+ */
+
+import fs = require("fs/promises");
+
+import * as express from 'express';
+const logger = require('lambda-log');
+
+const app = express()
+app.use(express.json())
+
+/**
+ * @capability schedule 0/5 * * * ? *
+ */
+export async function scheduledHealthSignal(){
+  // this method will be called every 5 minutes
+  logger.warn(`Health signal logged at ${Date.now()}`);
+}
+
+ app.listen(3000, async () => {
```

Comment:
For the most basic version, is any of the express stuff necessary since the scheduledHealthSignal is going to be in a different lambda

Reply:
I just wanted to make sure the service was actually running as a lambda so we could look at logs

Reply:
Also, we could definitely add more capabilities to make the app smarter (like an endpoint that returns the number of times the health signal has fired), but I felt like that mostly highlighted other aspects rather than scheduling